safe/gui/tools/help/peta_bencana_help.py

```diff
@@ -81,9 +82,9 @@ def content():
         'This tool requires a working internet connection and fetching '
         'data will consume your bandwidth.'))
     tips.add(m.Link(
-        'https://petajakarta.org/banjir/en/data/',
+        production_api['help_url'],
         text=tr(
-            'Downloaded data is copyright the PetaJakarta contributors'
+            'Downloaded data is copyright the PetaBencana contributors'
```

Comment:
I'm guessing it should be PetaBencana (it is from two words, Peta and Bencana).
Their docs API perhaps always use Petabencana, because it is using API docs generator, and auto-capitalize the project name or something.

I just assume it is PetaBencana since they even have a logo: PetaBencana.id. Or maybe even the official name is "PetaBencana.id", I don't know.